### PR TITLE
docs(client-go): update readme

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

File: client-go/README.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+`deis` is a command line utility used to interact with the [Deis](http://deis.io) open source PaaS.
 
-This is a WIP rewrite of the client in go.
+Please add any [issues](https://github.com/deis/deis/issues) you find with this software to
+the [Deis Project](https://github.com/deis/deis).
+
+## Installation
+
+Currently the only way to use the go version of the `deis` client is to build it yourself.
+To build the `deis` client, you need to have [go](https://golang.org/) and
```
Comment (bacongobbler) [inline client-go/README.md]: you also need Godep to build this client

**Comment by Joshua-Anderson** [comment]
@bacongobbler Done!

**Comment by bacongobbler** [comment]
lgtm :)

**Comment by mboersma** [comment]
LGTM.

**Pull request merged**